mmpy_bot/function.py

```diff
@@ -30,26 +35,21 @@ def __init__(
             self.siblings.append(function)
             function = function.function
 
-        self.function = function
+        self.function: Union[Callable, Function] = function
```

Comment:
```suggestion
        self.function = function
```
The type is already annotated just a few lines above. Also, given the function above, I believe `function` is guaranteed to just be a `Callable` here (if it was still a Function, the loop would have just triggered another time).